Dockerfile.test

```diff
@@ -1,6 +1,8 @@
 FROM python:3.7
 
+ENV PYLINT_PARAMS "--disable=fixme"
+
 ADD . /cli_client/
 RUN pip install /cli_client/
-RUN pip install pytest
-CMD python -m pytest /cli_client/tests/ -v
\ No newline at end of file
+RUN pip install pytest pylint
```

Comment:
consider extracting to requirements-test.txt

Reply:
Done https://github.com/random-development/cli-client/compare/06cc9b6aad163cd7a6a3276b8741bde907e4104d..113cee4c714aa3504a2946efb9f9d28c565b83c0